tests/examples/test_examples.py

```diff
@@ -28,8 +48,15 @@
     (os.path.join('tensorflow', 'tf1'), ['-P', 'steps=10']),
     ('xgboost', ['-P', 'learning_rate=0.3', '-P', 'colsample_bytree=0.8', '-P', 'subsample=0.9'])
 ])
-def test_mlflow_run_example(directory, params):
-    cli_run_list = [os.path.join(EXAMPLES_DIR, directory)] + params
+def test_mlflow_run_example(directory, params, tmpdir):
+    example_dir = os.path.join(EXAMPLES_DIR, directory)
+    tmpdir = os.path.join(tmpdir.strpath, directory)
```

Comment:
Nit: would name `tmpdir` something else to differentiate it from the pytest `tmpdir`

Reply:
Got it. I'll rename it to `tmp_example_dir`